src/control/lib/daos/pool_property.go

```diff
@@ -120,6 +120,31 @@ func PoolProperties() PoolPropertyMap {
 				valueMarshaler: numericMarshaler,
 			},
 		},
+		"svc_ops_entry_age": {
```

Comment:
Would be a good idea to add some test cases for this to [TestControl_PoolProperties](https://github.com/daos-stack/daos/blob/master/src/control/lib/daos/pool_property_test.go#L79). Looks like the `svc_ops_enabled` property could use some, too.

Reply:
I have a separate PR with some candidate unit test cases for these two new properties, https://github.com/daos-stack/daos/pull/13676